### PR TITLE
B5/output cleanup

### LINKED ISSUE #417: `iroh p2p peers` fix string concat to add p2p peer id correctly
missing the "/p2p" prefix

### PR DESCRIPTION
**Pull request by b5**
minor cleanups, better documentation of the lookup command.

Screen Shot 2022-10-25 at 10 47 11 PM

closes #417

### CODE REVIEW

**Comment by b5** [comment]
Also thought it was important to get a shoutout to @mxinden in the help text 😄

Screen Shot 2022-10-25 at 10 50 48 PM

File: iroh/src/doc.rs [hunk 1]
```diff
@@ -169,7 +169,20 @@ protocol. When provided with a peer ID, the address is looked up on the
...
+
+lookup will output 6 types of the info about the peer:
+
+  Peer ID             - the identifier of the peer
+  Agent Version       - software identifier similar to a web browser's user agent
+  Protocol Version    - broad identifier for the peer & it's network
+  Observed Addresses  - the peer's best guess at it's own public address(es)
+                        based on feedback from other peers
+  Listening Addresses - address this peer is listening for connections on
+  Protocols           - identifiers for protocols this peer speaks
```
Comment (b5) [inline iroh/src/doc.rs:182]: feel free to weigh in on these definitions. I do wish there was a section outlining these on the libp2p docs

**Comment by mxinden** [comment]
> Also thought it was important to get a shoutout to @mxinden in the help text smile

Very kind. Thank you @b5 :heart:

**Review by dignifiedquire — APPROVED** [review]

File: iroh/src/doc.rs (hunk 1 above)
Comment (mxinden) [reply iroh/src/doc.rs:182]: @b5 I would say the best we have is https://github.com/libp2p/specs/tree/master/identify#the-identify-message

**Pull request merged**